pep/app/authenticators/pep.ts

```diff
@@ -21,10 +25,12 @@ export default class PepAuthenticator extends BaseAuthenticator {
     async authenticate(username: string, password: string) {
         const params = serializeQueryParams({
             grant_type: 'password',
-            username,
-            password
+            UserName: username,
+            Password: password
+        });
+        const result = await this.ajax.request(`${ENV.authBaseUrl}/Authenticate?${params}`, {
+            headers: this.authenticationHeaders
         });
-        const result = await this.ajax.request(`Session/Login?${params}`);
         return result;
     }
```

Comment:
not sure if @adistasio talked to you about this yet, but apparently we are never doing OAuth now, so we'll need some to add some logic to the authenticator that automatically invalidates the local session when the returned session expires.

I'll most likely look somewhat similar to the `_scheduleAccessTokenRefresh` logic we have in our oauth-based authenticators (see CC or JRE), but instead of refreshing, itll just call `invalidate()` once the expire date is hit. This timer would most likely need to be kicked off both in `authenticate()` and `restore()`.

If we want to do this in a follow up PR, thats probably fine too.